R/mod_Plugins_UI.R

```diff
@@ -1,12 +1,14 @@
 #' Plugins Wrapper UI
 #'
 #' @inheritParams shared-params
+#' @returns Plugins UI elements, either as a tab (if there is only one plugin)
+#'   or in a drop-down list (for multiple plugins).
 #' @keywords internal
 mod_Plugins_UI <- function(id, lPlugins = NULL) {
   ns <- NS(id)
   if (!is.null(lPlugins)) {
     plugin_items <- purrr::imap(lPlugins, function(lPlugin, i) {
-      fnUI <- rlang::as_function(lPlugin$shiny$UI)
+      fnUI <- util_AsFunction(lPlugin$shiny$UI)
```

Comment:
so, once we fix up issue 1992 over in gsm, you'll use that function here, or is there still additional functionality that this method provides?

Reply:
TECHNICALLY I also have functionality in here to deal with a situation where a user sends a function in (without using plugin_Read() to generate their lPlugins entry). I don't know if that's important for sure. If it is, we should move it over to the central function (which I call inside `util_AsFunction()` right now).